lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -29,7 +29,7 @@ def define
         end
 
         next_node do
-          if calculator.ceremony_country == 'ireland'
+          if %w(ireland philippines).include?(calculator.ceremony_country)
```

Comment:
@leenagupte I would like to be explicit on line 32 about the country being migrated to 2 outcomes. 
I would like to have something like:

```ruby
if calculator.is_two_outcomes_country?
# ...
```

As there are a few countries already in fact-check [I have created a Trello card][1] to perform the refactor.

[1]: https://trello.com/c/V1T9dLEg/481-refactor-for-marriage-abroad-two-outcomes